odh-dashboard/base/kustomization.yaml

```diff
@@ -21,10 +21,11 @@ resources:
   - image-puller.clusterrolebinding.yaml
   - model-serving-role.yaml
   - model-serving-role-binding.yaml
+  - fetch-accelerators.rbac.yaml
 images:
 - name: odh-dashboard
   newName: quay.io/opendatahub/odh-dashboard
-  digest: sha256:4478d3830a30058fb87ed3e94dfdc615bf7517fddbd8490d6e2563263a0752fb
+  digest: sha256:a88ae7ba1b8c4ab2f0c3850b219ca1e031c73c1aaae79998358ae2a6850d7da5
```

Comment:
Out of date nightly -- need one more night to get the latest sha (this excludes the latest in `main`).

/hold